web/impact/impact/v1/views/user_list_view.py

```diff
@@ -100,6 +106,44 @@ def _invalid_keys(self, user_keys):
         for key in set(user_keys) - set(ALL_USER_RELATED_KEYS):
             self.errors.append(INVALID_KEY_ERROR.format(key))
 
+    def _user_results(self, limit, offset):
```

Comment:
Not a blocker, but it would be nice if this method had the same name here and in organization_list_view.py.  It would be even nicer if the redundant code was abstracted away into a shared method, but that might not be worth the effort.